src/database.ts

```diff
@@ -228,12 +327,22 @@ class Database extends ServiceObject {
    *   }
    * });
    */
-  close(callback) {
+  close(
+    callback?: SessionPoolCloseCallback
+  ): void | Promise<google.protobuf.IEmpty> {
```

Comment:
We have a different type listed in the overload for the Promise, not sure which one is correct but lets be consistent here.

Reply:
Looks like this still uses 2 different types for the Promise generic type.